src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -3251,19 +3196,28 @@ private Profile getCurrentProfile() {
 
     private void handleCommandLineOptions() {
 
-        if (parseArguments.isOptPresent("--help")) {
+        if (parseArguments.isOptPresent(AppConstants.HELP_OPTION_STRING)) {
             // -F is the ParseArguements internal option to pre-load command line options from a file
 
             String[] usage = { //
-                    "Usage: Starter [--help] [-F filename] [--server] [--first] [--login <login>] [--password <pass>] " + //
-                    "[" + LOAD_OPTION_STRING + " <dir>|<file> ] " + //
-                    "[--skipini] " + //
-                    "[" + NOLOGGING_OPTION_STRING + "] ", //
-                    "[" + INI_FILENAME_OPTION_STRING + " filename] [" + //
-                    CONTEST_PASSWORD_OPTION + " <pass>] [" + NO_GUI_OPTION_STRING + "] "+ //
-                    "[" + NOSTANDINGS_OPTION_STRING + "] "+ //
-                    "[" + REMOTE_SERVER_OPTION_STRING + " <remoteHostname> --proxyme]" + //
-                    "[" + MAIN_UI_OPTION + " classname]", // 
+                    "Usage: Starter " + //
+                    "[" + AppConstants.HELP_OPTION_STRING + "] " + //
+                    "[" + AppConstants.FILE_OPTION_STRING + " filename] " + //
+                    "[" + AppConstants.SERVER_OPTION_STRING + "] " + //
+                    "[" + AppConstants.FIRST_SERVER_OPTION_STRING + "] " + //
+                    "[" + AppConstants.LOGIN_OPTION_STRING + " <login> " + //
+                    "[" + AppConstants.PASSWORD_OPTION_STRING + " <pass> ", //
+                    "           " + 
+                    "[" + AppConstants.LOAD_OPTION_STRING + " <dir>|<file> ] " + //
+                    "[" + AppConstants.SKIP_INI_OPTION_STRING + "] " + //
+                    "[" + AppConstants.INI_FILENAME_OPTION_STRING + " INIfilename] " + //
+                    "[" + AppConstants.CONTEST_PASSWORD_OPTION_STRING + " <pass>] " + 
+                    "[" + AppConstants.REMOTE_SERVER_OPTION_STRING + " <remoteHostname> " + AppConstants.PROXY_ME_SERVER_KEY + "]" + //
+                    "[" + AppConstants.MAIN_UI_OPTION_STRING + " classname]", // 
+                    "           " +
+                    "[" + AppConstants.NOLOGGING_OPTION_STRING + "] " + //
+                    "[" + AppConstants.NO_GUI_OPTION_STRING + "] " + //
+                    "[" + AppConstants.NOSTANDINGS_OPTION_STRING + "] ", //
                     "", //
```

Comment:
I think there are at least two options missing here (they were missing in the original as well, so it's understandable):

   NO_CONNECTIONS_PANE_OPTION_STRING  and NO_LOGINS_PANE_OPTION_STRING

Reply:
The "**usage**" String array [] is a list of strings that are printed to show the command line usage.  Each element is printed using **System.out.println()**.  The idea of the "        " was to indent the second and third lines of options so they are formatted somewhat nicer instead of all at the left margin.  I can remove them and put everything on one line if that is preferred. (You'll notice that the lines preceding the "       " end in a comma, not a plus sign.

Note: I did notice a couple (LOGIN_OPTION_STRING and PASSWORD_OPTION_STRING) of the options in the usage string do not have a trailing ']'.  I have to add those too.

NO_CONNECTIONS_PANE_OPTION_STRING and NO_LOGINS_PANE_OPTION_STRING  were newer options, and I guess they were not added to the original usage message; I can add them.

Reply:
The usage message will display as follows:
```
Usage: Starter [--help] [-F filename] [--server] [--first] [--login <login>] [--password <pass>]
           [--load <dir>|<file> ] [--skipini] [--ini INIfilename] [--contestpassword <pass>]
           [--remoteServer <remoteHostname> server.proxyme] [--ui classname]
           [--nologging] [--nogui] [--nostandings] [--noconnectionspane] [--nologinspane]

```

Reply:
There are a couple of options that do not appear in the usage message.  One is not used anywhere (--id xxx) should it be removed?   The --nosave, --port xxx, and --team1 options are used, but not listed in the help.  Should they be?  I think --team1 is debugging shorthand option.

Reply:
I think that --port should be added
Do not include the following undocumented option (undocumented debugging options)
The --nosave (do not save anything to disk, only for a server) and 
--team1  shorthand for joe login on team 1

Reply:
New usage message changed to read as follows including --port option:

```
Usage: Starter [--help] [-F filename] [--server] [--first] [--login <login>] [--password <pass>]
           [--load <dir>|<file> ] [--skipini] [--ini INIfilename] [--contestpassword <pass>]
           [--remoteServer <remoteHostname> server.proxyme] [--ui classname] [--port <port>]
           [--nologging] [--nogui] [--nostandings] [--noconnectionspane] [--nologinspane]
```